Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Amazon.Lambda.TestTool.csproj

```diff
@@ -15,13 +15,17 @@
   </PropertyGroup>
 	
   <ItemGroup>
+      <PackageReference Include="Amazon.Lambda.RuntimeSupport" Version="1.11.0" />
       <PackageReference Include="Blazored.Modal" Version="7.3.1" />
 	  <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="8.0.11" /> 
-  </ItemGroup>	
+  </ItemGroup>
+
+  <ItemGroup>
+	  <None Include="$(OutputPath)Amazon.Lambda.RuntimeSupport.dll" Pack="true" PackagePath="content" Visible="false" />
```

Comment:
wasnt sure if this needed to be `content` or `lib` folder. was seeing this warning when doing pack command

```
The assembly 'content\Amazon.Lambda.RuntimeSupport.dll' is not inside the 'lib' folder and hence it won't be added as a reference when the package is installed into a project. Move it into the 'lib' folder if it needs to be referenced.
```

but in our case in we are referencing the dll as part of a command line argument in the aspire proof of concept , so not sure if it really matters

Reply:
I'm assuming this is only adding the current target framework (net8.0) version of RuntimeSupport. I believe we need to add all targets of Runtimesupport. @normj is my assumption correct?

Reply:
i remember he mentioned when he updated the test tool v2 something about to not worry about the other versions. but ill let him confirm